packages/select/src/Select.tsx

```diff
@@ -482,6 +503,13 @@ export default function Select({
           darkMode={darkMode}
           disabled={disabled}
           className={cx(
+            {
+              [largeLabelStyles]: size === Size.Large,
+              [css`
+                font-size: ${baseFontSize}px;
+                line-height: 20px;
```

Comment:
Can use `typeScales['body1' | 'body2'].lineHeight`

Reply:
the line-height for a label is always `20` when it's `default` which doesn't match whats in `typeScales`.  Maybe we should consider adding a label line height inside of typeScales?

```
body1: {
    fontSize: BaseFontSize.Body1,
    lineHeight: 20,
    labelLineHeignt: 20,
  } as const,
body2: {
    fontSize: BaseFontSize.Body2,
    lineHeight: 28,
    labelLineHeignt: 20,
  } as const,

```